datacats/docker.py

```diff
@@ -103,14 +103,15 @@ def _get_docker():
 
 class WebCommandError(Exception):
 
-    def __init__(self, command, container_id, logs):
+    def __init__(self, command, container_id, logs, ssh=False):
```

Comment:
I think that that approach can get really confusing after a while. The argument name is not very telling and will need a docstring/comment at least. Another thing is that it we differentiate between classes of errors with dedicated Exception classes elsewhere and here we differentiate between the two classes with True | False argument values.

I am voting for creating a dedicated type of exception for these issues:
```
class RemoteCommandError(WebCommandError):
      ... 
```
I think it is the pythonic way.